arch/arm64/boot/dts/rockchip/overlay/Makefile

```diff
@@ -2,6 +2,8 @@
 dtbo-$(CONFIG_ARCH_ROCKCHIP) += \
 	armsom-sige7-camera-imx415-4k.dtbo \
 	armsom-sige7-display-10hd.dtbo \
+	armsom-sige7-camera-ov13850-csi0.dtbo \
+	armsom-sige7-camera-ov13850-csi1.dtbo \
```

Comment:
Please add these two dtbo after `armsom-sige7-camera-imx415-4k.dtbo`, so that Makefile is sorted by the order of alphabet.